style.css

```diff
@@ -461,3 +453,182 @@ form {
   color: #5e6c84;
   border-color: #5e6c84;
 }
+
+@media (min-width: 768px) {
+  .first_project,
+  .second_project,
+  .third_project,
+  .forth_project {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border: 2px solid grey;
+    border-radius: 12px;
+    padding: 15px 35px;
+    background-color: #fff;
+    gap: 50px;
+    margin-bottom: 20px;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .navbar {
+    width: 100%;
+    background-color: white;
+    padding: 30px 154px;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    justify-content: space-between;
+    gap: 850px;
+  }
+
+  .dropdown {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 100px;
+    padding-bottom: 20px;
+    margin-top: 20px;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .Tonic {
+    width: 600px;
+    height: 400px;
+  }
+
+  .flex_list {
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+  }
+
+  .project_description {
+    width: 100%;
+    height: fit-content;
+    margin-bottom: 10px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .project_name {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-bottom: 20px;
+  }
+
+  .About_Section {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-around;
+    padding: 100px 120px;
+  }
+
+  .About_Section_description {
+    width: 400px;
+  }
+
+  .About_Section_flex {
+    max-width: 500px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    padding: 30px 0;
+  }
+
+  .About_Section_flexx {
+    max-width: 900px;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+    padding: 30px 0;
+  }
```

Comment:
- [ ] [ Optional] As when I switch the design to 817px then I noticed that the design is not responsive, this project focused on responsiveness so Please implement it. The image is overlapping on content  please fix it as well
### Your design
![your](https://user-images.githubusercontent.com/81727988/199730743-8490b617-0c59-4675-b5f6-c3a12bfa0767.png)

![Screenshot 2022-11-03 190313](https://user-images.githubusercontent.com/81727988/199731108-3bc897cf-4102-46ad-a52b-82b1b6e5a57d.png)